i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/applications-databases-oracle.md

```diff
@@ -185,31 +185,31 @@ Ce compte utilisateur doit avoir la permission de lecture sur les tables suivant
   - dba\_tablespaces
   - DBA\_MVIEW\_refresh\_times
   - DBA\_TABLESPACE\_USAGE\_METRICS
-  - v$sysstat
-  - v$sgastat
-  - v$parameter
-  - v$process
-  - v$session
-  - v$filestat
-  - v$log
-  - v$instance
-  - V$ASM\_DISKGROUP
-  - v$database\_block\_corruption
-  - v$tempstat
-  - v$rowcache
-  - v$system\_event
-  - v$recovery\_area\_usage
-  - v$librarycache
-  - v$sql\_monitor
-  - v$resource\_limit
-  - v$rman\_status
-  - v$backup
-  - v$rman\_status
-  - v$rollstat
-  - v$resource\_limit
-  - v$tablespace
-  - v$event\_name
-  - v$waitstat
+  - v_$sysstat
+  - v_$sgastat
+  - v_$parameter
+  - v_$process
+  - v_$session
+  - v_$filestat
+  - v_$log
+  - v_$instance
+  - V_$ASM\_DISKGROUP
```

Comment:
pourquoi un V majuscule ici et pas ailleurs?

Reply:
don't know, just applied what has been provided through the initial PR. From what I see [here](https://docs.oracle.com/en/database/oracle/oracle-database/12.2/ostmg/views-asm-info.html#GUID-82C92841-3812-4442-86E7-BA1496605685) it seems legit

Reply:
The view names are case insensitive so it can be lowercased for consistency